licenses/models.py

```diff
@@ -165,6 +168,20 @@ class LicenseRequest(LicenseTemplate, models.Model):
         default=False,
     )
 
+    def clean(self) -> None:
+        """Either the LR is a screen_board or the duration isn't null."""
+        if self.is_screen_board:
+            # it's a screen board, we are fine
+            self.duration = timedelta(
+                seconds=settings.SCREEN_BOARD_DURATION)
+            return super().clean()
+
+        if not self.duration:
+            raise ValidationError(
+                {'duration': _('Duration must not be null.')})
```

Comment:
Interesting, that this does not need another test to keep up with the coverage. I would still suggest to test this validation error message in a test.

Reply:
[ef8dfd0](https://github.com/gocept/ok_tools/pull/63/commits/ef8dfd00b879c39d5cc1cf01a73e8ee513f41913)